content/control-api/index.textile

```diff
@@ -0,0 +1,229 @@
+
+Using the Control API you can manage:
+
+* Your applications
+* Keys for an application
+* "Namespaces":/general/channel-rules-namespaces (for channel rules)
+* "Queues":/general/queues
+* "Reactor rules":/general/events
+
+Repetitive operations such as creating or updating apps, enumerating queues, and other tasks can be automated using the Control API.
+
+In order to use the Control API you need to first create an access token in the "Ably dashboard":https://ably.com/accounts/any. However, once you have done this you can use the Control API to manage your account without having to interact with the dashboard for many functions.
+
+h2(#oas3). OpenAPI Specification (OAS3)
+
+The OpenAPI Specification (OAS3) for this API can be found in the Ably OpenAPI Specifications "GitHub repository":https://github.com/ably/open-specs.
+
+h2(#authentication). Authentication
+
+Before you can use the Control API you need to create an access token for authentication. This is done in the Dashboard.
+
+In the "Ably dashboard":https://ably.com/accounts/any, on the top menu bar, select your account from the dropdown list and then select "My Access Tokens" from the menu:
+
+<a href="/images/screenshots/control-api/my-access-tokens-menu-item.png" target="_blank">
+  <img src="/images/screenshots/control-api/my-access-tokens-menu-item.png" style="width: 40%" alt="My Settings">
+</a>
+
+You are presented with the "My Access Tokens" area, where you can create tokens for use with the Control API:
+
+<a href="/images/screenshots/control-api/my-access-tokens.png" target="_blank">
+  <img src="/images/screenshots/control-api/my-access-tokens.png" style="width: 80%" alt="My Settings">
+</a>
+
+h3(#creating-access-token). Creating an access token
+
+To create a new token, click the "Create new access token" button. Then enter the required information into the dialog:
+
+1. Enter a friendly name for your token, so you'll remember what it's used for.
+2. Select the capabilities you wish the token to have, depending on your use case.
+3. Click the "Create" button to create the token.
+
+<a href="/images/screenshots/control-api/new-access-token.png" target="_blank">
+  <img src="/images/screenshots/control-api/new-access-token.png" style="width: 40%" alt="My Settings">
+</a>
+
+h3(#using-access-token). Using the access token
+
+From the "My access tokens" area you can click the "Copy Token" button, to copy the button to the clipboard.
+
+You use the access token to authenticate requests to the Control API. To do this, you supply the access token as a Bearer token in the Authorization header of the HTTP request. For example, in the folllowing Curl request replace `<YOUR_ACCESS_TOKEN>` with the token you have copied to the clipboard:
+
+```[sh]
+curl -H 'Authorization: Bearer <YOUR_ACCESS_TOKEN>' ...
+```
+
+h3(#obtain-token). Obtain token details using the Control API
+
+You can use the Control API to obtain your token details. This is shown in the following request:
```

Comment:
```suggestion
You can use the Control API to obtain information about a token, such as its capabilities and the user and account it is assigned to. This is shown in the following request:
```

Reply:
I used this more or less, but changed it to: "You can use the Control API to obtain information about your access token, such as its capabilities and the user and account it is assigned to."